Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: build_css help
+
+OUTPUT_FILE=lib/Themes/CustomCss.php
+TEMP_CSS=css/combined.css
+PLACEHOLDER=\/\* Generated custom CSS - Please run make build_css \*\/
+END_PLACEHOLDER=\/\* End of generated custom CSS \*\/
+
+help: ## Show this help message
+	@echo "Usage: make [target]"
+	@echo ""
+	@echo "Targets:"
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  %-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+build_css: ## Build the custom CSS file
+	@echo "Building CSS from files in $(CSS_DIR)..."
+	cat css/*.css | sed '/^$$/d; s/^/\t/' > $(TEMP_CSS)
+	sed -i "/$(PLACEHOLDER)/,/$(END_PLACEHOLDER)/{//!d}" $(OUTPUT_FILE)
```

Comment:
This command failed for me due to `$(OUTPUT_FILE)` (`lib/Themes/CustomCss.php`) not existing.